src/randdrop.md

```diff
@@ -197,12 +197,12 @@ junod tx wasm execute juno14tfzmpl6kw2e6lyqzsvjgcqu7a9jhsneu0u4nvdv6znulps6xufqw
 
 ### Request a randomness beacon from Nois
 
-The timestamp needs to be in microseconds
+The timestamp needs to be in nanoseconds since epoch
 
 ```sh
 junod tx wasm execute juno14tfzmpl6kw2e6lyqzsvjgcqu7a9jhsneu0u4nvdv6znulps6xufqwlw2xf \
- '{"rand_drop":{"random_beacon_after":1685188901000000000}}' \
- --amount 100ujunox \
+  '{"rand_drop":{"random_beacon_after":"1685188901000000000"}}' \
```

Comment:
Timestamp is a newtype for Uint64 which always needs to be represented as a String. Otherwise values > 53 bits lose precision.

Reply:
yeah indeed. I wrote this before actually running the randdrop. but entering a number instead of a string wouldn't even be accepted by the contract.